.github/workflows/zulip-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Zulip Tests
```

Comment:
The name should be just "build" because this name will show up in the build badge, and we want it short, just "build", like on Travis.

Reply:
sure